src/main/scala/devsearch/features/FeatureRecognizer.scala

```diff
@@ -100,19 +106,19 @@ case class CodePiecePosition(location: CodeFileLocation, line: Int) extends java
   override def toString = location.toString + ":" + line
 }
 
-case class CodeFile(language: String, location: CodeFileLocation, ast: AST) extends java.io.Serializable
+case class CodeFile(location: CodeFileLocation, ast: AST) extends java.io.Serializable
```

Comment:
I removed the language because it is not used by Feature, causing some confusing usages (see "unknown_language" and "Source" usages in old version).